addons/l10n_ca/data/tax_report.xml

```diff
@@ -668,178 +668,211 @@
+                                <field name="formula">CaSk3</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_ca_tr_pst_sk_4" model="account.report.line">
+                        <field name="name">4 - Tax Written Off on Bad Debts</field>
+                        <field name="code">CaSk4</field>
+                        <field name="hierarchy_level">5</field>
+                        <field name="expression_ids">
+                            <record id="l10n_ca_tr_pst_sk_4_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">CaSk4</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_ca_tr_pst_sk_5" model="account.report.line">
+                        <field name="name">5 - Other</field>
+                        <field name="code">CaSk5</field>
+                        <field name="hierarchy_level">5</field>
+                        <field name="expression_ids">
+                            <record id="l10n_ca_tr_pst_sk_5_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">CaSk5</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_ca_tr_pst_sk_6" model="account.report.line">
+                        <field name="name">6 - Tax Credits Recorded for this Period</field>
+                        <field name="code">CaSk6</field>
+                        <field name="hierarchy_level">3</field>
+                        <field name="aggregation_formula">CaSk2.balance + CaSk3.balance + CaSk4.balance + CaSk5.balance</field>
+                    </record>
+                    <record id="l10n_ca_tr_pst_sk_7" model="account.report.line">
+                        <field name="name">7 - Tax Credits</field>
+                        <field name="code">CaSk7</field>
+                        <field name="aggregation_formula">CaSk1.balance + CaSk6.balance</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ca_tr_pst_sk_step_2" model="account.report.line">
+                <field name="name">Step 2</field>
+                <field name="code">CaSkStep2</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="l10n_ca_tr_pst_sk_total_sales" model="account.report.line">
+                        <field name="name">Total Sales Before Taxes</field>
```

Comment:
Sorry, another change request :joy: we discussed this with Laura a bit further, and in the end we would like this line to be at the top of the report (before 'Step 1'). It's just that since we're not counting refunds under the total sales, it makes sense to put the total sales before all the computations of tax credits. Thanks!